src/main/sensors/battery.c

```diff
@@ -87,12 +100,12 @@ static void updateBatteryVoltage(void)
 
 void updateBattery(void)
 {
-    uint16_t vbatPreviousADC = vbatLatestADC;
+    uint16_t vbatPrevious = vbatLatest;
     updateBatteryVoltage();
-    uint16_t vbatMeasured = batteryAdcToVoltage(vbatLatestADC);
+    uint16_t vbatMeasured = vbatLatest;
```

Comment:
This is not needed anymore, just change the references to `vbatMeasured` below to `vbatLatest`.